src/app/components/fees-row/components/custom-fee-field.tsx

```diff
@@ -58,8 +58,8 @@ export function CustomFeeField({
           height="32px"
           name="fee"
           disabled={disableFeeSelection}
-          onChange={async (evt: FormEvent<HTMLInputElement>) => {
-            await helpers.setValue(evt.currentTarget.value);
+          onChange={(evt: FormEvent<HTMLInputElement>) => {
+            void helpers.setValue(evt.currentTarget.value);
```

Comment:
In the [Stacks UI removal PR](https://github.com/leather-wallet/extension/pull/4490/) we changed this to a promise as when moving to `styled.input` it started throwing a no floating promises warning:
```
Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator.eslint[@typescript-eslint/no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises)
```

This change instead marks it as `void` to get rid of this error but also stop firing sentry errors